### PR TITLE
GOV: write up policy on not updating req for CVEs in dependencies

### PR DESCRIPTION
**Pull request by tacaswell**
This comes up about every other month.

inspired by https://github.com/matplotlib/matplotlib/pull/28110

SC should sign off on this @timhoffm @dopplershift @efiring

### CODE REVIEW

File: doc/devel/min_dep_policy.rst [hunk 1]
```diff
@@ -76,6 +81,19 @@ In the case of GUI frameworks for which we rely on Python bindings being
 available, we will also drop support for bindings so old that they don't
 support any Python version that we support.
 
+Security Issues in Dependencies
+===============================
+
+In most cases we should not adjust the versions supported based on CVEs to our
+dependencies.  We are a library not an application and the version constraints
+on our dependencies indicate what will work (not what is wise to use).  Users
+and packagers can install newer versions of the dependencies their discretion
```
Comment (dopplershift) [inline doc/devel/min_dep_policy.rst:90]:
```suggestion
and packagers can install newer versions of the dependencies at their discretion
```

**Review by dopplershift — COMMENTED** [review]
Looks great other than the minimal grammar edit.

File: doc/devel/min_dep_policy.rst [hunk 2]
```diff
@@ -76,6 +81,19 @@ In the case of GUI frameworks for which we rely on Python bindings being
 available, we will also drop support for bindings so old that they don't
 support any Python version that we support.
 
+Security Issues in Dependencies
```
Comment (QuLogic) [inline doc/devel/min_dep_policy.rst:84]: Headings are in sentence case.
```suggestion
Security issues in dependencies
```

File: doc/devel/min_dep_policy.rst [hunk 3]
```diff
@@ -76,6 +81,19 @@ In the case of GUI frameworks for which we rely on Python bindings being
...
+===============================
+
+In most cases we should not adjust the versions supported based on CVEs to our
```
Comment (timhoffm) [inline doc/devel/min_dep_policy.rst:87]:
```suggestion
Generally, we do not adjust the supported versions of dependencies based on security vulnerabilities.

**Review by efiring — APPROVED** [review]
Looks good (with corrections as suggested by others).

**Review by dopplershift — APPROVED** [review]

**Review by timhoffm — APPROVED** [review]

**Pull request merged**